src/01-functions.js

```diff
@@ -9,7 +9,11 @@
  *  alwaysTrue();
  *  //> true
  */
-function alwaysTrue() {}
+function alwaysTrue(x) {
+  x = true;
```

Comment:
While this code works we are adding an input to the function.  How could we refactor this so that our function returns true with no parameters needing to be passed?